### PR TITLE
Round FM 36 Earning Period Amount

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs [hunk 1]
```diff
@@ -558,7 +558,289 @@ public void TestMultiplePriceEpisodes()
...
+                        PwayCode = 400,
+                        LearnDelInitialFundLineType = "Funding Line Type",
+                        LearnStartDate = DateTime.Today.AddDays(-10)
+                    },
+                    LearningDeliveryPeriodisedValues = new List<LearningDeliveryPeriodisedValues>
+                    {
+                        new LearningDeliveryPeriodisedValues
+                        {
+                            AttributeName = "MathEngOnProgPayment",
+                            Period1 = 100.00000900000m,
```
Comment (kuzinator) [inline src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs]: Can some numbers be changed to test edge cases?
100.00009000000m
100.00000444444m
100.00000555555m
etc

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs]: Done
